src/store/model/globaldata.ts

```diff
@@ -0,0 +1,329 @@
+import {Store} from './store';
+
+export const Globaldata: Store = {
+  currency: '€',
+  labels: {
+    inStock: {
+      container:
+        'body > main > div.main-container > div.container.mb-5 > div.row.align-items-lg-start.z-above-pattern > ck-product-cta-box > div > div.d-flex.mt-4 > div.w-100 > ck-addto-cart-wrapper > form > div > div.w-100.pl-3 > button > span',
+      text: ['Adicionar ao carrinho'],
+    },
+    outOfStock: {
+      container:
+        'body > main > div.main-container > div.container.mb-5 > div.row.align-items-lg-start.z-above-pattern > ck-product-cta-box > div > div.d-flex.mt-4 > div.w-100 > button',
+      text: ['Notifique-me quando estiver em stock'],
+    },
+  },
+  links: [
+    {
+      brand: 'test:brand',
+      model: 'test:model',
+      series: 'test:series',
+      url:
+        'https://www.globaldata.pt/grafica-asus-geforce-rtx-3080-ek-10gd6x-90yv0f60-m0nm00',
```

Comment:
```suggestion
        'https://www.globaldata.pt/drive-externa-dvdr-asus-sdrw-08d2s-u-lite-8x-branco-usb-20-90-dq0436-ua221kz',
```

Test model should have something that is usually in stock.